### PR TITLE
Migrate to Rust 2021

### PR DESCRIPTION
**Pull request by thomaseizinger**
Nothing to do because our code is already idiomatic!

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.55"
+channel = "1.56"
```
Comment (klochowicz) [inline rust-toolchain.toml:2]:
```suggestion
channel = "1.56.1"
```

I believe that that's the current stable

**Review by klochowicz — APPROVED** [review]
sweet! 👯‍♂️ 👯‍♀️

File: rust-toolchain.toml (hunk 1 above)
Comment (thomaseizinger) [reply rust-toolchain.toml:2]: It will auto-pick the highest patch release AFAIK.

**Pull request merged**